test/unit/edition/validation_test.rb

```diff
@@ -161,4 +161,12 @@ class Edition::ValidationTest < ActiveSupport::TestCase
     edition.supporting_organisations = [organisation_1]
     assert edition.valid?
   end
+
+  test 'should raise an exception when attempting to modify an edition of a locked document' do
```

Comment:
The description of this test indicates that we are testing that an edition belonging to a locked document is being modified, whereas in the test itself we're creating an edition then changing its document's `locked` status to true and then saving it, which isn't quite right.  I think we should create an edition belonging to a locked document and then attempt to modify the edition itself.